datasm/scripts/check_file_integrity.py

```diff
@@ -44,6 +45,7 @@ def main():
 
     if not input_path.exists() or not input_path.is_dir():
         con_message("error", f"Input directory does not exist or is not a directory")
+        log_message("info", f"DEBUG: Input directory does not exist or is not a directory")
```

Comment:
```suggestion
        log_message("debug", "Input directory does not exist or is not a directory")
```
Should this be a `debug` level message, rather than `info`?

Reply:
Same comment as line 36.